File: app/src/main/java/ua/regin/wallpapers/entity/ImageResponse.java
package ua.regin.wallpapers.entity;

import java.io.Serializable;

public class ImageResponse implements Serializable{

    private long id;
    private int height;
    private int width;
    private Image image;
    private String uploader;

    public long getId() {
        return id;
    }

    public int getHeight() {
        return height;
    }

    public int getWidth() {
        return width;
    }

    public Image getImage() {
        return image;
    }

    public String getUploader() {
        return uploader;
    }

}

File: app/src/main/java/ua/regin/wallpapers/entity/ImageResponse.kt
package ua.regin.wallpapers.entity;

import java.io.Serializable;

class ImageResponse : Serializable {

    var id: Long;
    var height: Int;
    var width: Int;
    var image: Image;
    var uploader: String;

    constructor(id: Long, height: Int, width: Int, image: Image, uploader: String) {
        this.id = id
        this.height = height
        this.width = width
        this.image = image
        this.uploader = uploader
    }
}
